package.json

```diff
@@ -1,96 +1,96 @@
-    "url": "https://github.com/s-yadav/react-number-format"
-  },
-  "bugs": {
-    "mail": "sudhanshuyadav2@gmail.com",
-    "url": "https://github.com/s-yadav/react-number-format/issues"
-  },
-  "keywords": [
-    "react-component",
-    "react",
-    "currency",
-    "input",
-    "number",
-    "number-format",
-    "mask"
-  ],
-  "scripts": {
-    "start": "webpack-dev-server --hot",
-    "bundle": "cross-env yarn compile && yarn bundle-dist && yarn test-build && yarn test-ts",
-    "bundle-dist": "cross-env NODE_ENV=production rollup -c rollup.config.js",
-    "compile": "cross-env NODE_ENV=production babel src --out-dir lib",
-    "test": "cross-env NODE_ENV=test karma start",
-    "test-build": "cross-env NODE_ENV=production TEST_BROWSER=ChromeHeadless karma start",
-    "test-ts": "yarn tsc -p typings",
-    "lint": "cross-env eslint src/**"
-  },
-  "devDependencies": {
-    "@babel/cli": "^7.0.0",
-    "@babel/core": "^7.0.1",
-    "@babel/plugin-proposal-class-properties": "^7.0.0",
-    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
-    "@babel/plugin-transform-flow-strip-types": "^7.0.0",
-    "@babel/preset-env": "^7.0.0",
-    "@babel/preset-react": "^7.0.0",
-    "@babel/register": "^7.0.0",
-    "@types/react": "^16.4.18",
-    "babel-eslint": "^9.0.0",
-    "babel-loader": "^8.0.2",
-    "babel-plugin-add-module-exports": "^1.0.0",
-    "babel-plugin-transform-object-assign": "^6.22.0",
-    "classnames": "^2.2.3",
-    "cross-env": "^3.1.4",
-    "enzyme": "^3.6.0",
-    "enzyme-adapter-react-16": "^1.5.0",
-    "eslint": "^5.16.0",
-    "eslint-loader": "^2.1.0",
-    "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-react": "^7.11.1",
-    "flow-bin": "0.54.1",
-    "jasmine": "^2.4.1",
-    "jasmine-core": "^2.4.1",
-    "karma": "^3.0.0",
-    "karma-chrome-launcher": "^2.2.0",
-    "karma-cli": "^1.0.1",
-    "karma-jasmine": "^0.3.7",
-    "karma-jasmine-html-reporter": "^0.2.0",
-    "karma-rollup-preprocessor": "^6.0.1",
-    "karma-sourcemap-loader": "^0.3.7",
-    "karma-spec-reporter": "^0.0.32",
-    "karma-webpack": "^3.0.5",
-    "material-ui": "^0.19.2",
-    "react": "^15.4.0 || ^16.0.0",
-    "react-dom": "^15.4.0 || ^16.0.0",
-    "react-router": "3",
-    "react-transform-hmr": "^1.0.4",
-    "rollup": "^1.11.3",
-    "rollup-plugin-babel": "^4.3.2",
-    "rollup-plugin-commonjs": "^9.3.4",
-    "rollup-plugin-filesize": "^6.0.1",
-    "rollup-plugin-license": "^0.8.1",
-    "rollup-plugin-node-resolve": "^4.2.3",
-    "rollup-plugin-replace": "^2.2.0",
-    "rollup-plugin-uglify": "^6.0.2",
-    "typescript": "^3.1.5",
-    "webpack": "^4.17.1",
-    "webpack-cli": "^3.1.0",
-    "webpack-dev-server": "^3.1.7"
-  },
-  "peerDependencies": {
-    "react": "^0.14 || ^15.0.0-rc || ^15.0.0 || ^16.0.0-rc || ^16.0.0",
-    "react-dom": "^0.14 || ^15.0.0-rc || ^15.0.0 || ^16.0.0-rc || ^16.0.0"
-  },
-  "dependencies": {
-    "prop-types": "^15.7.2"
-  }
-}
+    "name": "react-number-format",
```

Comment:
Can you maintain the tab spacing to 2. ?

Reply:
Done